### ISSUE TITLE
2.5.17 Default Admin Theme problem

### ISSUE THREAD

**Issue by eladnova** [ryancramerdesign/ProcessWire#905]
Install the new Config Module
Under Config>Core>Misc(9), choose AdminThemeReno as the Default Admin Theme
Admin resets to the much older skin

![config_ _processwire](https://cloud.githubusercontent.com/assets/1786476/6061604/0768221c-ad41-11e4-8fbd-5ac9e87fa8b1.png)

**Comment by mike-anthony** [comment]
So now it's really come out. The old theme popped up for me at some point, too. But not like this.

**Comment by ryancramerdesign** [comment]
I can't duplicate this one here yet. Though the old admin theme does serve as a fallback for when other admin theme modules may not be available in certain scenarios, but you shouldn't see it stick around for more than one page load. Are you seeing the old admin theme continue to appear even after reloading or clicking to another page?

**Comment by eladnova** [comment]
It's definitely sticking around :)

I had to reinstall the RenoTheme manually to enable it. It was working fine for the second time until I installed HannaCode which reset it back to the classic theme.

My next step was to set the Default Admin Theme to AdminThemeDefault in Config >Misc

That correctly applies the updated Admin theme. Strangely enough, setting AdminThemeReno as the default makes the admin resort back to the Classic theme.

Understand this would be very difficult to diagmose without access to the site which is currently running locally. Just thought I'd flag it in case anyone else see it.

**Comment by junofornow** [comment]
I had this problem as well, in combination with the Modules Manager plugin.
Every time I go to the Modules Manager the Old admin theme is applied, and it sticks, logout/login or new session doesn't matter.
Had to copy Reno from the source again, plus every time it happens the fix is to go to the Modules > Site, Uninstall Reno (not necessary to remove) and then Install it again from the Module page.

**Comment by outflux3** [comment]
@junofornow 
exact same problem happened to me last night, posted about it here:
https://processwire.com/talk/topic/9102-adminthemereno-disappeared/?p=87918

i couldn't get reno back, had to mess with the cache entry in the table;  
didn't try this though:
"Deleting Reno from the wire AdminThemes folder, and uploading it again, fixes it."
will try that if it happens again

**Comment by outflux3** [comment]
@junofornow solution didn't work for me...

**Comment by junofornow** [comment]
@outflux3 sorry... did it for me. Seems like a complex thing. I stayed away from the ModulesManager plugin and didn't have it anymore. But I don't know whether that's the culprit. or what caused it initially.

**Comment by wanze** [comment]
This issue is not related to ModulesManager. In my case a module completely dissapeard and I don't have ModulesManager installed.

**Comment by mike-anthony** [comment]
ModulesManager *may* have caused this. I had the issue once when doing something with ModulesManager, but it only appeared for one request, and then reset back to what it was before (Default). That said, I have ModulesManager installed in three different dev channels (.17, .18, and .19), and no issues have come up.

**Comment by outflux3** [comment]
yeah, when it happened last night, i wasn't using modules manager; so it can be narrowed down to the installation of modules, and the recreating of the caches db entries, because i was able to fix it by modifying the json there.

**Comment by mike-anthony** [comment]
Yeah, that would make sense.

**Comment by outflux3** [comment]
just happened again on a completely fresh install on a different server; was installing a module, and reno got burned

**Comment by eladnova** [comment]
Lost my Reno Theme this morning after being relatively stable for a while.

I was testing SwiftMailer and while installation went fine,  removing the Module set the theme back to the older version.

I did notice an earlier message regarding having two versions of Reno installed.

I don't have two versions of Reno installed. In my Modules folder I have a folder called AdminThemeReno and then a separate ZIP called AdminThemeReno-master.zip

I have AdminThemeReno set as the Default Admin Theme in the Config module. Possibly there's some conflict there?

**Comment by eladnova** [comment]
Running PW 2.5.22 BTW

**Comment by eladnova** [comment]
This is odd.
While the old theme was showing, I installed another module and Reno came back.
Uninstalling that Module caused the RenoTheme to disappear again.
